client/app/components/sections/ManageAvailability/ManageAvailabilityContainer.js

```diff
@@ -16,13 +16,15 @@ const ManageAvailabilityContainer = ({
   visibleMonth,
   hasChanges,
   saveInProgress,
+  saveFinished,
   reservedDays,
   blockedDays,
   sideWinderWrapper,
 }) =>
       r(ManageAvailability, {
         hasChanges,
         saveInProgress,
+        saveFinished,
```

Comment:
Btw. we do this quite much. What is actually removed from container props?
(I.e. could we do this passing of partial props with "...restProps" approach)
Anyway, just a thought.